<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimelineEdit</class>
 <widget class="QWidget" name="Timeline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timeline</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="toolBar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="topLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="record">
          <property name="icon">
           <iconset resource="../../../res/WorldEditor.qrc">
            <normaloff>:/Images/fontawesome/record.png</normaloff>:/Images/fontawesome/record.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>13</width>
            <height>13</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="begin">
          <property name="icon">
           <iconset resource="../../../res/WorldEditor.qrc">
            <normaloff>:/Images/fontawesome/fast-backward.png</normaloff>:/Images/fontawesome/fast-backward.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>13</width>
            <height>13</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="previous">
          <property name="icon">
           <iconset resource="../../../res/WorldEditor.qrc">
            <normaloff>:/Images/fontawesome/back.png</normaloff>:/Images/fontawesome/back.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>13</width>
            <height>13</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="play">
          <property name="icon">
           <iconset resource="../../../res/WorldEditor.qrc">
            <normaloff>:/Images/fontawesome/play.png</normaloff>:/Images/fontawesome/play.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>13</width>
            <height>13</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pause">
          <property name="icon">
           <iconset resource="../../../res/WorldEditor.qrc">
            <normaloff>:/Images/fontawesome/pause.png</normaloff>:/Images/fontawesome/pause.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>13</width>
            <height>13</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="next">
          <property name="icon">
           <iconset resource="../../../res/WorldEditor.qrc">
            <normaloff>:/Images/fontawesome/next.png</normaloff>:/Images/fontawesome/next.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>13</width>
            <height>13</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="end">
          <property name="icon">
           <iconset resource="../../../res/WorldEditor.qrc">
            <normaloff>:/Images/fontawesome/fast-forward.png</normaloff>:/Images/fontawesome/fast-forward.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>13</width>
            <height>13</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="clipBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="curve">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="icon">
           <iconset resource="../../../res/WorldEditor.qrc">
            <normaloff>:/Images/fontawesome/curve.png</normaloff>:/Images/fontawesome/curve.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>13</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="flatKey">
          <property name="text">
           <string>Flat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="breakKey">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Breaked</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deleteKey">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="time">
          <property name="text">
           <string>Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="timeEdit">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="value">
          <property name="text">
           <string>Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="valueEdit">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="KeyFrameEditor" name="widget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeyFrameEditor</class>
   <extends>QScrollArea</extends>
   <header>keyframeeditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../res/WorldEditor.qrc"/>
 </resources>
 <connections/>
</ui>
